scripts/deploy_new_chain.py

```diff
@@ -0,0 +1,25 @@
+from brownie import GasStationV2, accounts
+
+GITHUB_POKER="0x737760C760FfEc370F84861E4Be4AFF7093Ffa3f"
+OMNICHAIN_MULTISIG="0x9ff471f9f98f42e5151c7855fd1b5aa906b1af7e"
+def deploy_chain(account, chain):
+    keeper = OMNICHAIN_MULTISIG
+    GasStationV2.deploy(keeper, 60*60*8, {"from": account})
+    gs = GasStationV2[len(GasStationV2)-1]
```

Comment:
are you trying to get last entry in the list? in that case in python you can count negatively:
```suggestion
    gs = GasStationV2[-1]
```